chart/datadog-operator/templates/clusterrole.yaml

```diff
@@ -57,4 +61,165 @@ rules:
   - "list"
   - "get"
   - "watch"
+- apiGroups:
+  - autoscaling
+  resources:
+  - horizontalpodautoscalers
+  verbs:
+  - list
+  - watch
+## To fix: should be role not a clusterrole
```

Comment:
All those configmap are only accessed by the DCA and should be in the DCA namespace. So I think we can move it in the role.